ixp4xx_crypto: Fix compilation

This patch fixes following compile errors:

drivers/crypto/ixp4xx_crypto.c: In function 'ixp_module_init':
drivers/crypto/ixp4xx_crypto.c:1473:21: error: 'THIS_MODULE' undeclared (first use in this function)
drivers/crypto/ixp4xx_crypto.c:1473:21: note: each undeclared identifier is reported only once for each function it appears in
drivers/crypto/ixp4xx_crypto.c: At top level:
drivers/crypto/ixp4xx_crypto.c:1502:16: error: expected declaration specifiers or '...' before string constant
drivers/crypto/ixp4xx_crypto.c:1503:15: error: expected declaration specifiers or '...' before string constant
drivers/crypto/ixp4xx_crypto.c:1504:20: error: expected declaration specifiers or '...' before string constant

---
 drivers/crypto/ixp4xx_crypto.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/crypto/ixp4xx_crypto.c
+++ b/drivers/crypto/ixp4xx_crypto.c
@@ -18,6 +18,7 @@
 #include <linux/interrupt.h>
 #include <linux/spinlock.h>
 #include <linux/gfp.h>
+#include <linux/module.h>
 
 #include <crypto/ctr.h>
 #include <crypto/des.h>
